### ISSUE TITLE
Accept-Encoding: gzip shouldn't be set

### ISSUE THREAD

**Issue by mtt87** [contentful/contentful.js#74]
![screen shot 2016-05-27 at 12 21 58](https://cloud.githubusercontent.com/assets/2150927/15606745/fbf4f56c-2405-11e6-9fe4-8fe60a3f4926.png)

Chrome 50.0.2661.102

You shouldn't set the `Accept-Encoding` header as it's automatically set by the browser afaik.

Seems like it's this line in the code of the compiled browser library `contentful.js`
```js
headers = headers || {};
headers['Authorization'] = 'Bearer ' + accessToken;
// headers['Accept-Encoding'] = 'gzip';
```

**Comment by trodrigues** [comment]
Ah, yes, we've set this because of usage on node.js. We'll make sure to not set it when running on a browser. Thanks for the report!

**Comment by trodrigues** [comment]
Hi there! This should be fixed on the latest version, 3.3.12. Let me know if you still see the problem.

**Comment by shivawahi** [comment]
My images coming from Contentful, doesn't seems to be gzip compressed. Is there any setting that I need to make?
